BC_Food_Banks/BC_Food_Banks.py

```diff
@@ -0,0 +1,76 @@
+import os
+import sys
+from datetime import datetime
+import re
+import numpy as np
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+from pymongo import MongoClient, errors
+from tqdm import tqdm
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+from shared_code.utils import (
+    check_similarity, locate_potential_duplicate,
+    insert_services, client
+)
+from shared_code.base_scraper import BaseScraper
+
+class MFB_Scraper(BaseScraper):
```

Comment:
why is the class name same for both scrapers? Standard for python class names is CamelCase so BCFoodBanksScraper is more appropriate. (The same BaseScraper is named)

https://www.python.org/dev/peps/pep-0008/#class-names